pandas/core/indexes/multi.py

```diff
@@ -757,32 +769,66 @@ def set_levels(self, levels, level=None, inplace=False, verify_integrity=True):
         Examples
         --------
         >>> idx = pd.MultiIndex.from_tuples([(1, 'one'), (1, 'two'),
-                                            (2, 'one'), (2, 'two')],
+                                            (2, 'one'), (2, 'two'),
+                                            (3, 'one'), (3, 'two')],
                                             names=['foo', 'bar'])
-        >>> idx.set_levels([['a', 'b'], [1, 2]])
+        >>> idx
+        MultiIndex([('1', one),
+                    ('1', two),
+                    ('2', one),
+                    ('2', two),
+                    ('3', one),
+                    ('3', two)],
+                   names=['foo', 'bar'])
+        >>> idx.set_levels([['a', 'b', 'c'], [1, 2]])
         MultiIndex([('a', 1),
                     ('a', 2),
                     ('b', 1),
-                    ('b', 2)],
+                    ('b', 2),
+                    ('c', 1),
+                    ('c', 2)],
                    names=['foo', 'bar'])
-        >>> idx.set_levels(['a', 'b'], level=0)
+        >>> idx.set_levels(['a', 'b', 'c'], level=0)
         MultiIndex([('a', 'one'),
                     ('a', 'two'),
                     ('b', 'one'),
-                    ('b', 'two')],
+                    ('b', 'two'),
+                    ('c', 'one'),
+                    ('c', 'two')],
                    names=['foo', 'bar'])
         >>> idx.set_levels(['a', 'b'], level='bar')
         MultiIndex([(1, 'a'),
                     (1, 'b'),
                     (2, 'a'),
-                    (2, 'b')],
+                    (2, 'b'),
+                    (3, 'a'),
+                    (3, 'b')],
+                   names=['foo', 'bar'])
+
+        ``set_levels()`` passes values into the levels attribute that
```

Comment:
```suggestion
        If any of the levels passed to `set_levels` exceeds the existing length, all of the values from that argument will be stored though truncated in the MultiIndex output
```

Reply:
Thanks @WillAyd for the feedback - I've been attempting to express the concept through examples and all, but your explanation is very succinct and neat! Will incorporate those changes you've suggested. :)